kinta-workflows/src/main/kotlin/com/dailymotion/kinta/workflows/builtin/playstore/UpdatePlayStoreListings.kt

```diff
@@ -1,6 +1,7 @@
 package com.dailymotion.kinta.workflows.builtin.playstore
 
 import com.dailymotion.kinta.integration.googleplay.GooglePlayIntegration
+import com.dailymotion.kinta.integration.googleplay.LocalMetadataHelper
```

Comment:
Needed ?

Reply:
Yes it is.
LocalMetaHelper has just moved to kinta-lib because GetPlayStoreMetadata has moved as well